rules/0.17-allow-new-players.py

```diff
@@ -0,0 +1,29 @@
+import util
+
+# This value set to 0 until we figure out a good way to prevent dummy account abuse
+max_start_bonus = 0
+
+def should_allow(pr):
+  bonuses = pr.get_new_bonuses_or_raise()
+  
+  if len(bonuses) > 1:
+    raise Exception('Only one new player can be added in a PR')
+  elif len(bonuses) < 1:
+    raise Exception('Empty diff?')
+  
+  (points_user, points_name, points_change) = bonuses[0]
+  
+  if points_user in util.users():
+    raise Exception('Cannot create an existing user')
```

Comment:
This isn't going to work: when this runs on the master branch it will succeed (because points_user will be a new user) will be new, but when it runs on the PR branch it will fail (because points_user will not be a new user on the PR branch).  You don't need this check, though, since you're already requiring that this PR add a file called `initial`, which will already exist for any current user.

See https://github.com/jeffkaufman/nomic/pull/243 for an example of this rule failing an example PR to add a user: https://travis-ci.com/jeffkaufman/nomic/builds/101126028

```
...
Running rule rules/0.17-allow-new-players.py
PASS: new-players

...

Running rule rules/0.17-allow-new-players.py
Traceback (most recent call last):
  File "validate.py", line 84, in determine_if_mergeable
    if fn(pr_copy):
  File "rules/0.17-allow-new-players.py", line 17, in should_allow
    raise Exception('Cannot create an existing user')
Exception: Cannot create an existing user
  rules/0.17-allow-new-players.py: Cannot create an existing user
...
```

The PASS is on master, then the FAIL is on the branch.